### DISCUSSION TITLE
Beer Time Android app

### DISCUSSION THREAD

**Discussion by jonathanmccarthy** [MonoGame/MonoGame#8431] [Showcase]
I have released my first ever Android app on the Google Play Store.
Written using monogame (of course).

[![Google Play Store](https://github.com/user-attachments/assets/60c17ec1-d827-46bc-ab0f-e76baccbddaf)](https://play.google.com/store/apps/details?id=beer.time.abbot&pli=1)

Ever missed your bus or train because you were finishing your beer?
Beer Time can ensure you never miss that last ride home!
Set the timer to the amount of time you have before your vehicle is due.
Press play and watch your virtual beer go down. Simply match your drinking pace to the same level and you will catch that train AND finish your beer!
Just remember to drink responsibly!

There is a short demo video on Youtube:

[![Beer Time](https://github.com/user-attachments/assets/c35a61b5-3d06-4791-8570-35febec62464)](http://www.youtube.com/watch?v=ycG4-PVyDF4  "Beer Time")

And here are a few screen captures:

| screenshots | screenshots |
| ------------- | ------------- |
| ![SS1](https://github.com/user-attachments/assets/ed692031-4866-4dd6-ab14-61941f081321)  | ![SS2](https://github.com/user-attachments/assets/b55dfdaf-f685-43ca-a18e-3361c21785e8)  |
| ![SS3](https://github.com/user-attachments/assets/bccf06da-4b40-4724-b98c-61123089e799)  | ![SS4](https://github.com/user-attachments/assets/0376e92a-82a2-4be1-9521-f385e3f9ef81)  |

Thank you monogame for being an amazing tool!

**Comment by stromkos** [comment]
Be sure to conserve battery life by setting `TargetElapsedTime` to say 250 ms or 4 FPS and enable fast touch input, or 64 ms 15 FPS to still be responsive to input.

**Reply by jonathanmccarthy** [reply]
Thanks. I will look into that.
